modin/core/execution/dispatching/factories/test/test_dispatcher.py

```diff
@@ -113,8 +113,8 @@ def test_factory_switch():
 
 def test_engine_wrong_factory():
     with pytest.raises(FactoryNotFoundError):
-        with _switch_value(StorageFormat, "Pyarrow"):
-            with _switch_value(Engine, "Dask"):
+        with _switch_value(Engine, "Dask"):
+            with _switch_value(StorageFormat, "Pyarrow"):
```

Comment:
Due to the fact that the engine and the engine are not initialized at the same time, another error began to appear here, namely that `ExperimentalPyarrowOnRay` factory cannot be initialized because the experimental mode is not enabled.

Reply:
Should we put an inline comment here?